crates/project_panel/src/project_panel.rs

```diff
@@ -1759,6 +1767,10 @@ impl ProjectPanel {
                     .indent_level(depth)
                     .indent_step_size(px(settings.indent_size))
                     .selected(is_selected)
+                    .when_some(canonical_path, |this, path| {
+                        this.end_slot::<Icon>(Icon::new(IconName::ArrowRight))
+                            .tooltip(move |cx| Tooltip::text(path.clone(), cx))
```

Comment:
Not insisting, but it might be confusing for certain people still, to see an arrow and some path.
VSCode also adds a small suffix doc like this: `$PATH · Symbolic Link`
Should we also?

Reply:
Yeah It would be a good addition if any just to match the experience plus its an easy add.

I was concerned about the size of the tooltip so I was exploring matching the behavior of the tooltip on the tab in the pane.

![tab-ex](https://github.com/zed-industries/zed/assets/12102857/c6e9ee08-94be-4381-a7d7-502b0dd4f510)

The repo I'm using is very nested so it might be a non-issue for most cases but what do you think?